fixtures/swift-perform-checksum-once/tests/bindings/test_perform_checksum_once.swift

```diff
@@ -0,0 +1,16 @@
+import CoreFoundation
+import swift_perform_checksum_once
+
+let entity = newStruct()
+// The checksum is performed the first time a Rust method is called
+_ = fn0(arg: entity)
+
+// So we measure time between second and 10,000th calls
+_ = fn1(arg: entity)
+let startTime = CFAbsoluteTimeGetCurrent()
+for _ in 0..<10000 {
+  _ = fn2(arg: entity)
+}
+
+let elapsedTime = CFAbsoluteTimeGetCurrent() - startTime
```

Comment:
The PR seems fine but I'm skeptical of this test - every single time a test tries to assert something about wall time it ends up failing in some overloaded CI environment. TBH it's not clear a fixture is really needed here, although maybe having it under "benchmarks" and arranging for it not to be run by CI would make sense - WDYT?

Reply:
It was me suggesting a timed test -- but I had the same concerns as you @mhammond the moment I reviewed this PR by my dear colleague Mathias, it was a bad suggestion by me! But still it was good to measure, we got the results, a 220 x speedup :D 

Converting the fixture to a benchmark makes sense.

Reply:
@mhammond we would love to get this PR merged ASAP - and @matiasbzurovski is just now learning Rust, so would take some more time for him to convert the fixture to a benchmark. We might be able to do it later though.

So I suggest we update the PR with just the actual *fix*, and a benchmark can be added later.

WDYT?

Reply:
SGTM

Reply:
Removed the fixture and left the fix only.